src/shared/components/Button/Button.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface Button {
```

Comment:
Create separate file in`Button` folder called `types.ts`. You should keep all the types there.
This should be:
 ```
 export type Props = { 
   className: string;
   onClick: React.MouseEventHandler<HTMLButtonElement>;
   buttonText: string;
 }
 ```

Reply:
Done